lib/definitions/length.js

```diff
@@ -40,6 +40,13 @@ imperial = {
     }
   , to_anchor: 1/12
   }
+, yd: {
+    name: {
+      singular: 'Yard'
+    , plural: 'Yards'  
+    }
+  , to_anchor: 1/0.3333
```

Comment:
This seems like a mistake: the anchor is meters, so I was expecting `to_anchor: 0.9144`.